### ISSUE TITLE
Image Gallery is loading all images even when Lazy Loading is enabled

### ISSUE THREAD

**Issue by DavidGMiles** [elementor/elementor#14463]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Watching image loading in the network tab of DevTools reveals that all images are now being loaded by the gallery irrelevant of the setting for lazyloading - meaning that previously heavy galleries that only partially loaded are now loading all the images

## Steps to reproduce

Add a gallery to a page - add 100 images to the gallery - view the page - see all images are loaded in the network tab of dev tools

Now switch on lazy loading - view the page - see all images are STILL loaded in the network tab of dev tools 

This behaviour is not what it used to be - and has resulted in performant pages becoming very slow

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by sproutedweb** [comment]
Confirmed!

**Comment by fruitygod** [comment]
Yeah confirmed. Tried several settings, still every image is showing up. I am trying to optimize a portfolio for a photographer and this makes it nearly impossible to achieve.

**Comment by PeteNorfolk** [comment]
Discovered this the other day and it was mentioned on the Elementor Facebook Group.

Had to get rid of this all together, spent 2 days trying to come up with something else. Crazy!

**Comment by DavidGMiles** [comment]
Is this being looked at - this is a massive performance hit

**Comment by DavidGMiles** [comment]
Is this being looked at - this is a massive performance hit

**Comment by shilo-ey** [comment]
Hey, thanks for reporting.

I can't seem to reproduce this issue. can you please share a websitete that this issue is occurring in?

thanks

**Comment by DavidGMiles** [comment]
Not now no - I've had to roll the sites back - had you asked sooner then yes I could have - 29 days for anyone to even look at the ticket is a poor showing on Elementors part - the report was corroborated by four others but Elementor hasn't even looked at the raised issue for over 4 weeks

**Comment by PeteNorfolk** [comment]
And this the very reason we are rebuilding our sites and moving away from Elementor.
Dreadful!
